fix( mysql-workbench): Fix keyring support by adding IPC_Lock

Crow-Control opened the pull request:
**Description**
IPC_Lock is required for keyring to function it looks like, according to the container docs:
https://hub.docker.com/r/linuxserver/mysql-workbench

⚒️ Fixes  #2446 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [X] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**
<!--
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-->

**📃 Notes:**
<!-- Please enter any other relevant information here -->

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

Commit:
fix( mysql-workbench): Fix keyring support by adding IPC_Lock

Commit:
Update Chart.yaml

hugalafutro commented:
I updated the app to 8.0.28_1.0.11 but it didn't have the IPC_LOCK cap (Nor did it on freshly created app). I apologize If I misunderstand the update process and the change will be pushed later. For now I added the cap to the pod manually.

Crow-Control commented:
> I updated the app to 8.0.28_1.0.11 but it didn't have the IPC_LOCK cap (Nor did it on freshly created app). I apologize If I misunderstand the update process and the change will be pushed later. For now I added the cap to the pod manually.

How did you verify this?

hugalafutro commented:
I tried to add a db connection in the updated app and got the same error I reported in #2446. After adding the cap manually I was able to connect to a db as the keychain worked and allowed me to save the password. Sorry if I'm missing something, I'm more of a docker guy (truenas scale/truecharts is my first experience with k8s).

Crow-Control commented:
Okey, thanks, this highlights am (unrelated) bug in common then, i'll make note of that bug and will get it done before the end of summer :)

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
